### DISCUSSION TITLE
Overtraining causes models to have a strong english accent

### DISCUSSION THREAD

**Discussion by vertexgamer** [voicepaw/so-vits-svc-fork#414] [General]
Overtraining causes models to have a strong english accent, any idea on how to avoid this?

**Comment by vertexgamer** [comment]
UPDATE: The issue is not caused by overtraining, but happens to new models too (started happening since 3.10)

**Comment by xieyumc** [comment]
maybe this is Hubert issues, try to use a different Hubert?

**Comment by 34j** [comment]
Already resolved
